internal/provider/resource_access_generic.go

```diff
@@ -72,7 +72,7 @@ type genericJAASAccessModel struct {
 // ConfigValidators sets validators for the resource.
 func (r *genericJAASAccessResource) ConfigValidators(ctx context.Context) []resource.ConfigValidator {
 	return []resource.ConfigValidator{
-		RequiresJAASValidator{Client: r.client},
+		NewRequiresJAASValidator(r.client),
```

Comment:
todo:
missing from resource_access_jaas_model, there is a list of these at line 20.
```
var _ resource.ResourceWithConfigValidators = &jaasAccessModelResource{}
```